README.md

```diff
@@ -10,8 +10,13 @@ PySTAC is a library for working with [SpatialTemporal Asset Catalog](https://sta
 
 ## Installation
 
-PySTAC requires Python>=3.6. Support for Python>=3.10 should be considered experimental
-until further notice.
+PySTAC requires Python>=3.7. This project follows the recommendations of
+[NEP-29](https://numpy.org/neps/nep-0029-deprecation_policy.html) in deprecating support
+for Python versions. This means that users can expect support for Python 3.7 to be
+removed in next major or minor release after Dec 26, 2021.
```

Comment:
Maybe better to say a planned date to remove from the main branch.

Reply:
Yeah, that makes sense. Our current branching strategy doesn't have a way of doing patch releases without including features, so it would really be included in the next release of any kind.

Reply:
Fixed in c6bf9510e82aebcf800ef73769856dbde34f12ba

Reply:
And I directly use main:

 I do a [copybara](https://github.com/google/copybara) sync automatically every week.   That's basically git pull + patches + remove anything non-functional except the license file.